banking/klarna_kosma_integration/doctype/bank_reconciliation_tool_beta/bank_reconciliation_tool_beta.py

```diff
@@ -1105,7 +1179,7 @@ def get_pi_matching_query(
 			ConstantColumn("Purchase Invoice").as_("doctype"),
 			purchase_invoice.name,
 			purchase_invoice.paid_amount,
-			purchase_invoice.bill_no.as_("reference_no"),
+			purchase_invoice[reference_field or "name"].as_("reference_no"),
```

Comment:
Shouldn't this still default to `bill_no` instead of `name`? Might be an unpleasant change for the users, otherwise.